---
layout: paper
title: "Deconvoluting complex correlates of COVID-19 severity with a multi-omic pandemic tracking strategy"
image: /images/papers/2022-08-08-parikh-stanford-covid-ancestry.png
authors: NP Victoria, GI Alexander, Jimenez-Morales D, Gorzynski JE, Jong HND, Liu X, Roque J, Cepeda-Espinoza VP, Osoegawa K, Hughes C, Sutton SC, Youlton N, Joshi R, Amar D, Tanigawa Y, Russo D, Wong J, Lauzon JT, Edelson J, Montserrat DM, Kwon Y, Rubinacci S, Delaneau O, Cappello L, Kim J, Shoura MJ, Raja AN, Watson N, Hammond N, Spiteri E, Mallempati KC, Montero-Martín G, Christle J, Kim J, Kirillova A, Seo K, Huang Y, Zhao C, Moreno-Grau S, Hershman SG, Dalton KP, Zhen J, Kamm J, Bhatt KD, Isakova A, Morri M, Ranganath T, Blish CA, Rogers AJ, Nadeau K, Yang S, Blomkalns A, O’Hara R, Neff NF, DeBoever C, Szalma S, Wheeler MT, Farh K, Schroth GP, Febbo P, deSouza F, Fernandez-Vina M, Kistler A, Palacios J, Pinsky BA, Bustamante CD, Rivas MA, Ashley EA
year: 2022
ref: "Parikh et al. 2022. Nature Communications"
journal: "Nature Communications 13:5107"
pdf: /pdfs/papers/2022-parikh-stanford-covid-ancestry.pdf
supplement: /pdfs/papers/2022-parikh-stanford-covid-ancestry-si.pdf
doi: 10.1038/s41467-022-32397-8
---

### Abstract
The SARS-CoV-2 pandemic has differentially impacted populations of varied race, ethnicity and socioeconomic status. Admixture mapping and local ancestry inference represent powerful tools to examine genetic risk within multi-ancestry genomes independent of these confounding social constructs. Here, we leverage a pandemic tracking strategy in which we sequence viral and host genomes and transcriptomes from 1,327 nasopharyngeal swab residuals and integrate them with digital phenotypes from electronic health records. We demonstrate over-representation of individuals possessing Oceanian and Indigenous American ancestry in SARS-CoV-2 positive populations. Genome-wide-association disaggregated by admixture mapping reveals regions of chromosomes 5 and 14 associated with COVID19 severity within African and Oceanic local ancestries, respectively, independent of overall ancestry fraction. Phylodynamic tracking of consensus viral genomes reveals no association with disease severity or inferred ancestry. We further present summary data from a multi-omic investigation of human-leukocyte-antigen (HLA) typing, nasopharyngeal microbiome and human transcriptomics that reveal metagenomic and HLA associations with severe COVID19 infection. This work demonstrates the power of multi-omic pandemic tracking and genomic analyses to reveal distinct epidemiologic, genetic and biological associations for those at the highest risk.
